demisto_sdk/__main__.py

```diff
@@ -2265,6 +2265,57 @@ def error_code(config, **kwargs):
     sys.exit(result)
 
 
+@main.command(
+    name='create-content-graph',
+    help='create content graph'
+)
+@click.help_option(
+    '-h', '--help'
+)
+@click.option('-nd', '--no-use-docker', is_flag=True, help="Use docker to run the content graph")
```

Comment:
The name (with 'no') might be misleading.

Reply:
changed